linera-core/Cargo.toml

```diff
@@ -28,7 +28,7 @@ test = [
     "tokio/parking_lot"
 ]
 rocksdb = ["linera-views/rocksdb", "linera-storage/rocksdb"]
-aws = ["linera-views/aws", "linera-storage/aws"]
+aws = ["linera-views/dynamodb", "linera-storage/aws"]
```

Comment:
Why are you using `/dynamodb`?
If I am correct, we are not using a feature `dynamodb` but we do use `aws`.

Reply:
When I was factoring our feature configurations into config aliases, I took the opportunity to rename `aws` to `dynamodb` in the places where it was talking only about DynamoDB — since `aws` could be talking about Amazon RDS, or one of the many other databases AWS provides.  It seemed in line with the other database feature flags, which refer to the database name itself rather than the database's provider.  In crates more removed from the database, like `linera-core` or `linera-service`, I left it as `aws`, since I'm not sure those are referring to the database rather than all features required to run on an AWS cluster.

Reply:
The thing is that `DynamoDb` is only available on AWS. Therefore the feature `aws` though different in looks from `scylladb` and `rocksdb` corresponds relatively well to what it is we want.
So, I am afraid that renaming will introduce some confusion without really improving the functionality. But some people may have different opinion.

Reply:
DynamoDB also [supports running locally](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html) (mostly for testing) or [in Kubernetes](https://artifacthub.io/packages/helm/keyporttech/dynamodb).

Isn't it also true that ScyllaDB is only available in environments with a ScyllaDB instance running?

I assume that `linera-views/aws` can actually run in any environment running DynamoDB, and not in an environment (even an AWS environment) without DynamoDB running — hence the renaming.  I think I would agree with the `aws` name if AWS were sufficient to run the `aws` feature — e.g. if we set up a DynamoDB instance ourselves in `linera-views`.